templates/web/buckinghamshire/contact/who.html

```diff
@@ -0,0 +1,48 @@
+[% IF problem %]
+<h4>Topic:</h4>
+
+[% IF field_errors.dest %]
+<div class="form-error">[% field_errors.dest %]</div>
+[% END %]
+
+<div class="checkbox-group">
+    <input name="dest" id="dest_rules" type="radio" value="rules" class="required"[% IF dest AND dest == 'rules' %] checked[% END %]>
+    <label class="inline" for="dest_rules">This [% update ? 'update' : 'report' %] is abusive, contains personal information, or similar</label>
+</div>
+
+<div class="checkbox-group">
+    <input name="dest" id="dest_update" type="radio" value="update" class="required"[% IF dest AND dest == 'update' %] checked[% END %]>
+    <label class="inline" for="dest_update">I wish to report it has not been fixed yet</label>
+</div>
+
+<div class="checkbox-group">
+    <input name="dest" id="dest_council" type="radio" value="council" class="required"[% IF dest AND dest == 'council' %] checked[% END %]>
+    <label class="inline" for="dest_council">I wish to make a new report</label>
+</div>
+
+<div id="dest-error"[% IF NOT field_errors.not_for_us %] class="hidden"[% END %]>
+  <div class="form-error">You can only use this form to report inappropriate content</div>
```

Comment:
Because this is the standard error class you seem to get an odd interaction between the standard JS validation and the extra contact thing if
 1: do not click any of the destination options
 2: submit form and the standard "This field is required" message is display
 3: select one of the invalid options - message appears but not the header:

![screenshot 2018-12-19 at 15 06 17](https://user-images.githubusercontent.com/5310/50228366-aa199f00-039f-11e9-96dd-f342e8954560.png)

I suspect the extra validation needs to undo the bit of the standard validation that hides elements with a `form-error` class.

Reply:
You're missing the CSS update in that screenshot, I think, which would at least put that message in a big red box? :) I'll try and see what's happening.
![screen shot 2018-12-19 at 16 52 38](https://user-images.githubusercontent.com/154364/50234855-827e0300-03ae-11e9-95c9-ba86c8a496fa.png)